docs/using/recipe_code/oracledb.dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM jupyter/base-notebook
+
+## Install java, javac and alien
+USER root
+RUN apt-get update --yes && \
+    apt-get install --yes --no-install-recommends software-properties-common && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*  && \
+    apt-get update --yes && \
```

Comment:
These lines can be removed, to make installation faster.

Reply:
👌 Changed in latest version.